nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -149,6 +183,19 @@ a.anchor-link {
 .rendered_html p {
   text-align: inherit;
 }
+::-webkit-scrollbar
+{
+  width: 6px;
+  height: 6px;
+}
```

Comment:
It is overriding stuff on the slideshow... and this scroll is there specifically to scroll the slide.
Btw, this gives me the opportunity to make the scroll closer to other elements... otherwise you get the "system" view which it is not fitting at all the the slide feeling.